tests/test_pycdf_istp.py

```diff
@@ -649,10 +649,11 @@ def testFillvalEpoch16NoDefaultFill(self):
             'FILLVAL', (1.0, 2.0),
             type=spacepy.pycdf.const.CDF_EPOCH16)
         errs = spacepy.pycdf.istp.VariableChecks.fillval(v)
-        self.assertEqual(
-            'FILLVAL [1. 2.] (9999-12-13 23:59:59.999999),'  
-            ' should be (-1e+31, -1e+31) (9999-12-31 23:59:59.999999)' 
-            ' for variable type CDF_EPOCH16.', errs[0])
+        self.assertRegex(
+            errs[0],
+            'FILLVAL \\[\\s?1.\\s+2.\\] \\(9999-12-13 23:59:59.999999\\),'
+            ' should be \\(-1e\\+31, -1e\\+31\\)'
+            ' \\(9999-12-31 23:59:59.999999\\) for variable type CDF_EPOCH16.')
```

Comment:
This is also very straightforward, and looks fine, I think it is a pretty easy change in order to get the tests passing again.